third_party/misc/brython_time.py

```diff
@@ -0,0 +1,406 @@
+# from browser import window
```

Comment:
Write something from scratch is definitely possible, since strftime implementation is simple, I was just not sure using third party vs writing from scratch in terms of maintainability, but looks like that's not the case for this one.